src/bin/krillup.rs

```diff
@@ -1,89 +1,161 @@
-            KrillUpMode::Prepare { config } => {
-                let properties_manager = match PropertiesManager::create(
-                    &config.storage_uri,
-                    config.use_history_cache,
-                ) {
-                    Ok(mgr) => mgr,
-                    Err(e) => {
-                        eprintln!("*** Error Preparing Data Migration ***");
-                        eprintln!("{}", e);
-                        eprintln!();
-                        eprintln!("Note that your server data has NOT been modified. Do not upgrade krill itself yet!");
-                        eprintln!("If you did upgrade krill, then downgrade it to your previous installed version.");
-                        ::std::process::exit(1);
-                    }
-                };
-
-                match prepare_upgrade_data_migrations(
-                    UpgradeMode::PrepareOnly,
-                    &config,
-                    &properties_manager,
-                ) {
-                    Err(e) => {
-                        eprintln!("*** Error Preparing Data Migration ***");
-                        eprintln!("{}", e);
-                        eprintln!();
-                        eprintln!("Note that your server data has NOT been modified. Do not upgrade krill itself yet!");
-                        eprintln!("If you did upgrade krill, then downgrade it to your previous installed version.");
-                        ::std::process::exit(1);
-                    }
-                    Ok(opt) => match opt {
-                        None => {
-                            info!("No update needed");
-                        }
-                        Some(report) => {
-                            let from = report.versions().from();
-                            let to = report.versions().to();
-                            if report.data_migration() {
-                                info!("Prepared and verified upgrade from {} to {}.", from, to,);
-                            } else {
-                                info!("No preparation is needed for the upgrade from {} to {}.", from, to)
-                            }
-                        }
-                    },
+    };
+    config.log_level = LevelFilter::Info;
+    config.log_type = LogType::Stderr;
+
+    match options.command {
+        Command::Prepare(_prepare) => {
+            let properties_manager = match PropertiesManager::create(
+                &config.storage_uri,
+                config.use_history_cache,
+            ) {
+                Ok(mgr) => mgr,
+                Err(e) => {
+                    eprintln!("*** Error Preparing Data Migration ***");
+                    eprintln!("{}", e);
+                    eprintln!();
+                    eprintln!(
+                        "Note that your server data has NOT been modified. \
+                         Do not upgrade krill"
+                    );
+                    eprintln!("itself yet!");
```

Comment:
This looks a bit strange.

Reply:
Yeah, but it is sort of correct (if you want to have each line in its own println command).